napari/components/viewer_model.py

```diff
@@ -366,12 +381,25 @@ def add_image(self, image, *args, **kwargs):
         layer : :class:`napari.layers.Image`
             The newly-created image layer.
         """
-        layer = layers.Image(image, *args, **kwargs)
+        layer = layers.Image(
+            image,
+            metadata=metadata,
+            multichannel=multichannel,
+            colormap=colormap,
+            clim=clim,
+            clim_range=clim_range,
+            interpolation=interpolation,
+            opacity=opacity,
+            blending=blending,
+            visible=visible,
+            name=name,
+            **kwargs,
```

Comment:
sometimes we have `**kwargs` - sometimes we don't - see `add_points`. are they just missing in `add_points`? do we want them everywhere? i think we can drop them, but curious why we'd want them

Reply:
This PR merely ensures that layers and their respective convenience methods have the same signatures and parameter docs. As to why the `Image`, `Labels`, and `Volume` layers contain unused, undocumented `**kwargs` should be addressed elsewhere.

Reply:
makes sense